spec/controllers/v0/institutions_controller_spec.rb

```diff
@@ -123,6 +123,12 @@
       expect(JSON.parse(response.body)['data'].count).to eq(1)
     end
 
+    it 'filters by vet_tec_provider schools' do
+      create(:institution, :vet_tec_provider, vet_tec_provider: true)
```

Comment:
Suggestion: create another record where `vet_tec_provider: false` to show that they are not returned:
```suggestion
      create(:institution, :vet_tec_provider, vet_tec_provider: true)
      create(:institution, :vet_tec_provider)
```

Reply:
Added the other record to the test.